go.mod

```diff
@@ -241,6 +242,8 @@ replace github.com/deckhouse/deckhouse/dhctl => ./dhctl
 
 replace github.com/deckhouse/deckhouse/go_lib/cloud-data => ./go_lib/cloud-data
 
+replace github.com/deckhouse/deckhouse/go_lib/cni-cilium => ./go_lib/cni-cilium
```

Comment:
Storing module specific stuff in go lib is not what we usually do. If this is module specific, store it in the module directory (in any folder you like, e.g., `apis`).

Reply:
We use these structures in both the module and the controller.

Reply:
We need the structs in the controller image. We did it in analogy with cloud-data (@name212 's advice).